src/widgets/event/ui/TicketInfo.tsx

```
@@ -33,10 +37,41 @@ const TicketInfo = ({ eventId }: { eventId: number }) => {
-    const orderInfo = { ticketId, eventId, ticketCnt };
-    navigate('/payment/ticket-option-response', { state: orderInfo });
+    try {
+      const res = await readTicketOptions(ticketId);
+      if (res.isSuccess && res.result.length > 0) {
+        const orderInfo = { ticketId, eventId, ticketCnt };
+        navigate('/payment/ticket-option-response', { state: orderInfo });
+      } else {
+        // 옵션 없음 → 바로 결제
+        handleDirectOrder(ticketId, eventId, ticketCnt);
+      }
+    } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)